packages/firebase_ml_vision/android/src/main/java/io/flutter/plugins/firebasemlvision/ImageLabeler.java

```diff
@@ -0,0 +1,90 @@
+package io.flutter.plugins.firebasemlvision;
```

Comment:
This was a name change from `LabelDetector.java`. Changes can be seen [here](https://github.com/flutter/plugins/pull/1367/commits/d7e0bb819c779ad573760a8b1ae22138d9802835).